recipe/meta.yaml

```diff
@@ -156,17 +138,17 @@ outputs:
         - {{ compiler('cuda') }}  # [cuda_compiler_version != "None"]
         - swig
         - cmake
-        - autotools_clang_conda   # [win]
+        - libgomp                 # [linux]
+        - llvm-openmp             # [osx]
       host:
         - python
         - pip
-        - numpy =1.14.*
+        - numpy
         - libfaiss ={{ version }}=*_{{ faiss_proc_type }}
         - libblas
         - liblapack
-        - libgomp                 # [linux]
-        - llvm-openmp             # [osx or win]
       run:
+        - python
```

Comment:
I added this after getting warnings that this was missing, but that warning has merely transformed into:
```
WARNING (faiss): interpreter (Python) package conda-forge::python-3.7.8-hffdb5ce_3_cpython in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
```

Not sure what the correct approach is here.

Reply:
The way you have it now is correct. `conda-build` has no special handling for "runtime packages that impose run requirements without linking", meaning it treats `python` just like a library for which it would expect to see its shared objects being linked. Ergo, it's a known false positive we ignore.
(We may want to fix this in `conda-build` at some point, ideally with something more general that `if dep == `python` ;).)

Reply:
Cool thanks.